tests/zeroex/optimism/zeroex_api_optimism_fills_test.sql

```diff
@@ -0,0 +1,12 @@
+WITH unit_tests as
+(SELECT case when test.maker_token_address = actual.maker_token 
+                and test.taker_token_amount = actual.taker_token_amount 
+                and test.taker_token_address = actual.taker_token
+then True else False end as test
+FROM {{ ref('zeroex_api_optimism_fills') }} actual
+JOIN {{ ref('zeroex_api_optimism_fills_sample') }} test 
+    ON test.tx_hash = actual.tx_hash AND test.evt_index = actual.evt_index
+)
+select count(case when test = false then 1 else null end)/count(*) as pct_mismatch, count(*) as count_rows
```

Comment:
is `count(null)`=0 or =1 here? cuz looks like the tests failed due to this (same issue for all tests)
<img width="1061" alt="image" src="https://user-images.githubusercontent.com/22208403/207664193-b2e97c4a-66e6-4f76-a2f3-1205cd593bb6.png">